src/lwc/edaSetupHealthCheckRun/edaSetupHealthCheckRun.html

```diff
@@ -0,0 +1,37 @@
+<template>
+    <lightning-card title={labelReference.stgEDASettingsHealthCheckTitle}>
+        <div class="slds-grid slds-wrap">
+            <div class="slds-col 
+                        slds-size--1-of-1 
+                        slds-p-horizontal_small 
+                        slds-p-vertical_small
+                        slds-text-body_regular">
+                <p>{labelReference.stgHealthCheckDescription}</p>
+            </div>
+
+            <div class="slds-col 
+                        slds-size--1-of-1 
+                        slds-medium-size--1-of-2 
+                        slds-large-size--1-of-2 
```

Comment:
What is driving size 1 of 1 here?

Reply:
If viewed on a small device, size 1 of 1 would make the button stack below the last run date instead of crowding it against the text. Should I removed it?